extensions/http/pom.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.mule.extensions</groupId>
+        <artifactId>mule-extensions</artifactId>
+        <version>4.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-module-http-ext</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Mule HTTP Extension</name>
+    <description>A Mule extension that provides HTTP server and client functionality</description>
+
+    <dependencies>
+        <dependency>
```

Comment:
all mule dependencies should be provided. Otherwise they'll be repackaged inside the plugin